### ISSUE TITLE
[$10][MSFT-251] "Save Filter" button foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2903]
**Steps to Reproduce**
1. Go to: https://www.topcoder.com/challenges
2. Click on "Filters" button in the top right
3. Click on "keyword" dropdown and select any keyword in list dropdown to "Save filter" button enable
4. Right click > Inspect Elements 
5. Check the "TSave filter" button

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change Background Color to #0672EF Or you can change text color to #01135A. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #FFFFFF
Background Color: #0681FF
Contrast Ratio: 3.75:1
=> Therefor, "Topcoder handle" name foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F30

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096006 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096006 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096006 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@Oanh-and-only-Oanh @ThomasKranitsas can you please guide my how to reproduce this?

seems something different in this

**Comment by PrakashDurlabhji** [comment]
@Oanh-and-only-Oanh @veshu  PR https://github.com/topcoder-platform/community-app/pull/2928

**Comment by Dara-K** [comment]
Please use #006AD7 for the button color.

**Comment by PrakashDurlabhji** [comment]
@Dara-K @codeMinter code updatd according to above.
PR #2928

**Comment by PrakashDurlabhji** [comment]
@codeMinter PR updated #2928

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096737 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096737 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter PR updated

**Comment by codeMinter** [comment]
@veshu @sushilshinde PR closed as this is NOT AN ISSUE anymore in base repo. So closed PR.

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter anything if i can get for efforts?

**Comment by veshu** [comment]
@PrakashDurlabhji  Per reviewer the fix was not acceptable https://github.com/topcoder-platform/community-app/pull/2928#issuecomment-515567309 so we can not proceed the payment for this.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
